iphone/Classes/TiUIWebView.m

```diff
@@ -715,8 +715,8 @@ -(CGFloat)contentHeightForWidth:(CGFloat)value
     CGRect oldBounds = [[self webview] bounds];
     BOOL oldVal = webview.scalesPageToFit;
     [webview setScalesPageToFit:NO];
-    [webview setBounds:CGRectMake(0, 0, 10, 1)];
-    CGFloat ret = [webview sizeThatFits:CGSizeMake(10, 1)].height;
+    [webview setBounds:CGRectMake(0, 0, value, 1)];
```

Comment:
There is a test-case in TIMOB-14993 that also suggested this solution instead:
```objc


-(CGFloat)contentHeightForWidth:(CGFloat)value
{
     CGRect oldBounds = [[self webview] bounds];
     [webview setBounds:CGRectMake(0, 0, MAX(value,10), 1)];
     CGFloat result = [[webview stringByEvaluatingJavaScriptFromString:@"document.height"] floatValue];
     [webview setBounds:oldBounds];
     return result;
}
```
Thoughts?